pkg/virt-handler/vm.go

```diff
@@ -2710,6 +2787,13 @@ func (d *VirtualMachineController) vmUpdateHelperDefault(origVMI *v1.VirtualMach
 		return err
 	}
 
+	if vmi.IsCPUDedicated() && vmi.Spec.Domain.CPU.IsolateEmulatorThread {
```

Comment:
What if we're dealing with a VM with dedicated CPUs but without isolation of emulator thread?
Don't we still need to move all of the tasks to the housekeeping cgroup, but without isolating the emulator thread?

This would mean that VMs with dedicated CPUs would still not really get what they requested